### ISSUE TITLE
Onboarding card selected state should be keyboard accessible

### ISSUE THREAD

**Issue by kjohnson** [impress-org/givewp#4988]
## Details

Using the outline as the selected state doesn't provide an accessible keyboard experience. Instead, implement a checkmark icon as the selected state and re-purpose the outline as an active/focus state.

## Acceptance Criteria

-   [x] Selected card inputs are marked with a checkmark icon.
-   [x] Focusing a card input adds an outline state.
